keel-ec2-plugin/src/main/kotlin/com/netflix/spinnaker/keel/api/ec2/ClusterSpec.kt

```diff
@@ -51,15 +51,16 @@ private fun ClusterSpec.resolveLaunchConfiguration(region: SubnetAwareRegionSpec
     instanceType = checkNotNull(overrides[region.name]?.launchConfiguration?.instanceType
       ?: defaults.launchConfiguration?.instanceType),
     ebsOptimized = checkNotNull(overrides[region.name]?.launchConfiguration?.ebsOptimized
-      ?: defaults.launchConfiguration?.ebsOptimized),
+      ?: defaults.launchConfiguration?.ebsOptimized
+      ?: LaunchConfiguration.DEFAULT_EBS_OPTIMIZED),
     iamRole = checkNotNull(overrides[region.name]?.launchConfiguration?.iamRole
       ?: defaults.launchConfiguration?.iamRole
       ?: LaunchConfiguration.defaultIamRoleFor(moniker.app)),
```

Comment:
Side-question: we're still using the hard-coded, Netflix-specific IAM role template as a default when nothing is specified in the spec. Is there such a thing as "default IAM role name" for an account in clouddriver?

Reply:
Follow-up: it doesn't look like it. There's actually a template configured for the default IAM role in the clouddriver config file, but it's not exposed anywhere in the API that I could find.